test_heap.py

```diff
@@ -0,0 +1,20 @@
+# _*_ coding=utf-8 _*_
+
+HEAP_LIST = [39, 67, 23, 1, 765, 3, 2, 87]
+
+
+def test_push():
+    from binary_heap import BinHeap
+    bins = BinHeap()
+    for i in HEAP_LIST:
+        bins.push(i)
+    assert bins.heap[0] == 765
```

Comment:
How else could you test this? Currently this test only proves that one item is in one spot, but are you sure that the rest of the heap is correctly constructed?